index.html

```diff
@@ -48,6 +48,28 @@
 </head>
 
 <body>
+  <!-- <div> <input id=annoying-warning type=checkbox title="hide note">  -->
```

Comment:
@nigelmegitt I think this will need to be reviewed and approved by the TTWG once the PR is stable, right?

Reply:
Of course. I'm just getting the ball rolling.

Reply:
Yes, TTWG owns this document and will need to process the proposals.